whisper_jax/pipeline.py

```diff
@@ -87,6 +87,7 @@ def __init__(
             self.checkpoint,
             _do_init=False,
             dtype=self.dtype,
+            input_shape=(1, self.feature_extractor.feature_size, 3000),
```

Comment:
Thanks @gleb-sternharz! This fix would enable the large-v3 model for the `FlaxWhisperPipline`, but not the `model` + `processor` API (since the `input_shape` would still be wrong there). I've merged a fix that addresses both: https://github.com/sanchit-gandhi/whisper-jax/pull/160#discussion_r1407792409